AI_Graphics/AI_Graphics/Setting/PostProcessingSettings.cs

```diff
@@ -163,6 +163,10 @@ public enum Antialiasing {
         internal Transform VolumeTriggerSetting {
             get => _postProcessLayer.volumeTrigger;
         }
+        
+        internal string BloomTexturePath {
```

Comment:
what uses BloomTexturePath?

Reply:
It will set bloom parameter's texture path value without exposing paramBloom.

Reply:
Just don't see it used anywhere in the commit. If you already have subsequent change that uses it, feel free to resolve the conversation, if not, please remove.